lib/scripts/seed_test_db.ts

```diff
@@ -55,8 +55,8 @@ async function seedUsers(users: UserJSON[]): Promise<Prisma.BatchPayload> {
 
     const newUser: User = {
       id: user.id,
-      firstName: user.first_name,
-      lastName: user.last_name,
+      firstName: user.first_name || 'test',
+      lastName: user.last_name || 'user',
```

Comment:
can you add some sort of identifier here so that we can tell what user is what? we are already passing in the index so we can just do the following
```suggestion
      firstName: user.first_name || `Test-${index}`,
      lastName: user.last_name || `User-${index}`,
```